README.md

```diff
@@ -13,35 +13,118 @@ and [JaCoCo](https://github.com/jacoco/jacoco).
 
 ## Quickstart
 
-### Add repository
+### Add Maven Central repository
 
 For `settings.gradle`
 ```
 pluginManagement {
     repositories {
         // ... other dependencies
-        maven { url 'https://maven.pkg.jetbrains.space/public/p/jb-coverage/maven' }
+        mavenCentral()
     }
 }
 ```
 ### Apply plugin to project
-
+#### Applying plugins with the plugins DSL
 For `build.gradle.kts`
 ```
 plugins {
-     id("kotlinx-kover") version "0.3.0"
+     id("org.jetbrains.kotlinx.kover") version "0.3.0"
 }
 ```
 For `build.gradle`
 ```
 plugins {
-    id 'kotlinx-kover' version '0.3.0'
+    id 'org.jetbrains.kotlinx.kover' version '0.3.0'
+}
+```
+
+#### Legacy Plugin Application: applying plugins with the buildscript block
+For `build.gradle.kts`
+```
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath("org.jetbrains.kotlinx:kover:0.3.0")
+    }
+}
+
+apply(plugin = "org.jetbrains.kotlinx.kover")
+```
+
+For `build.gradle`
+```
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.jetbrains.kotlinx:kover:0.3.0'
+    }
 }
+  
+apply plugin: 'org.jetbrains.kotlinx.kover'    
 ```
 
 The plugin automatically inserted into `check` tasks pipeline and collects coverage during test run,
 verifying set validation rules and optionally producing XML or HTML reports.
 
+### Multi-module projects
+There is currently no full support for multi-module projects, you need to apply a plugin for each module.
+You can add the plugin to the `build.gradle` or `build.gradle.kts` files in each module, or add this code to the root module
+
+*Cross-module tests are not supported in reports and validation yet. For each test, only the classpath belonging to the current module is taken.*
+
+#### apply plugin for all modules 
+For `build.gradle.kts`
+```
+plugins {
+     id("org.jetbrains.kotlinx.kover") version "0.3.0"
+}
+
+
+allprojects {
+    apply(plugin = "org.jetbrains.kotlinx.kover")
+}
+```
+
+For `build.gradle`
+```
+plugins {
+    id 'org.jetbrains.kotlinx.kover' version '0.3.0'
+}
+
+
+allprojects {
+    apply plugin: 'org.jetbrains.kotlinx.kover'
+}
+```
+
+#### apply plugin only for submodules
+For `build.gradle.kts`
+```
+plugins {
+     id("org.jetbrains.kotlinx.kover") version "0.3.0" apply false
+}
+
+subprojects {
+    apply(plugin = "org.jetbrains.kotlinx.kover")
+}
+```
+For `build.gradle`
+```
+plugins {
+    id 'org.jetbrains.kotlinx.kover' version '0.3.0' apply(false)
+}
+
+subprojects {
+    apply plugin: 'org.jetbrains.kotlinx.kover'
```

Comment:
This haven't worked for me for some strange reason

Reply:
Error?